### PR TITLE
[UX] Treat inverse_reading_order as a proper setting

### PR DESCRIPTION
**Pull request by Frenzie**
This is supported by ReaderPaging and ReaderRolling, but the menu entry was only shown in ReaderPaging.

Hold now sets the global default.

Fixes <https://github.com/koreader/koreader/issues/4983>.
Fixes <https://github.com/koreader/koreader/issues/4089>.

### CODE REVIEW

File: frontend/apps/reader/modules/readerpaging.lua [hunk 1]
```diff
@@ -240,6 +246,21 @@ function ReaderPaging:addToMainMenu(menu_items)
             self.inverse_reading_order = not self.inverse_reading_order
             self:setupTapTouchZones()
         end,
+        hold_callback = function(touchmenu_instance)
+            self.inverse_reading_order = not self.inverse_reading_order
+            
+            self:setupTapTouchZones()
```
Comment (poire-z) [inline frontend/apps/reader/modules/readerpaging.lua]: Is toggling it and calling self:setupTapTouchZones() (here and in readerrolling) before the ok_callback right?
Either not at all (if settings as default should not apply to the current document), or in the ok_callback if it should?

Comment (Frenzie) [reply frontend/apps/reader/modules/readerpaging.lua]: Good point. I think not at all would be consistent with how it works elsewhere, and also with the logic behind text string.

**Comment by Frenzie** [comment]
Looks like this:

Screenshot_20190830_123823
Screenshot_20190830_123832

**Comment by poire-z** [comment]
Might find its way nearer to (or inside) "Taps & gestures" ?

And if we are going to support RTL in documents, we might soon think about supporting it in the UI.
So, may be that setting should apply as a whole, including when browsing menus?

Page/menu turning seems like a behavioral habit, so I don't know if people can naturally switch their page turning reflexes when reading English or Arabic PDFs... and when browsing menus.

**Comment by Frenzie** [comment]
There's no "going to". This has been there for years. It should either be like this or not at all, imho. ;-)

But indeed, it should go inside taps & gestures.

**Comment by Frenzie** [comment]
Although, not necessarily. This is a document-specific setting. Taps & gestures aren't.

**Comment by Frenzie** [comment]
How about next to its little friend in navigation? (I don't really know when that was added, lol.)

Screenshot_20190830_125918

**Comment by poire-z** [comment]
Yep, that looks fine (the other comes from fixing #4446).
Might be nice to add some `help_text=` to it/these :)

**Comment by Frenzie** [comment]
Isn't help_text also a hold_callback?

**Comment by Frenzie** [comment]
I'll merge this one, and leave help text for another commit/PR.

**Comment by poire-z** [comment]
> Isn't help_text also a hold_callback?

Oh, right :)

**Comment by Frenzie** [comment]
Could potentially still be useful on the other one. :-)

**Comment by hius07** [comment]
As we now have a new `Page turns` menu (https://github.com/koreader/koreader/pull/7572), isn't it a better place for `Invert page turns` items?

**Comment by NiLuJe** [comment]
There's also a case to be made for moving those to "Navigation", maybe?

But, in any case, agreed that they should be together ;).

**Comment by poire-z** [comment]
"Invert page turns" is a bit different than all the others found in "Taps and gestures" as it is a per-document setting, and should probably be easily'er accessible for people who switch between arabic and english books. Moving it in `Tap&Gestures> Page turns>` would make it one tap further than what it currently is when in `Navigation>` (but not too top level, as 99% of our users only read LTR and don't need that item). But as I'm not one of these people, I wouldn't mind much :)

**Comment by hius07** [comment]
> it is a per-document setting

I don't have a device with buttons, but am I right that
https://github.com/koreader/koreader/blob/617ed2c0782944efad48e16063a88bc785aa2e61/frontend/ui/elements/common_settings_menu_table.lua#L428
is applicable to the file browser page turns?

Secondly, is it a good idea to extend disabling swipes onto the file browser?
https://github.com/koreader/koreader/pull/7572#issuecomment-828381154

**Comment by poire-z** [comment]
> is applicable to the file browser page turns?

Looks like it is, it's handled very low level in device/generic/device.lua Device:init().

> Secondly, is it a good idea to extend disabling swipes onto the file browser?

Answering that there (not that I have much to say :), to not mix issues.

**Comment by NiLuJe** [comment]
Oh, right, it's for LTR/RTL swaps, right. Yeah, probably ought to stay put, then ;).

**Pull request merged**